collector/installation.go

```diff
@@ -125,24 +113,52 @@ func (i Installation) Collect(c *CollectorOpts) interface{} {
 	return i
 }
 
-func (i Installation) GetUid(uid string, c *CollectorOpts) (string, bool) {
-	if uid != "" {
-		log.Debugf("  Using Existing Uid: %s", uid)
+func (i *Installation) GetVersion(c *CollectorOpts) {
+	version, err := c.Client.Setting.ByID(SERVER_VERSION_SETTING)
+	if err != nil {
+		log.Errorf("  Failed to get setting %s err=%s", SERVER_VERSION_SETTING, err)
```

Comment:
Extra spacing in lots of places

Reply:
Extra spaces on error messages is my fault. Addressed.